Modules/_sqlite/cursor.c

```diff
@@ -426,6 +426,7 @@ PyObject* _pysqlite_query_execute(pysqlite_Cursor* self, int multiple, PyObject*
         }
     } else {
         /* execute() */
+        second_argument = _PyTuple_Empty;
         if (!PyArg_ParseTuple(args, "O|O", &operation, &second_argument)) {
             goto error;
         }
```

Comment:
What is going on here?  If multiple, then there is guaranteed to be something stuck in the pointer at second_argument, but otherwise ... there might be anyway?  If that is really the API, I would appreciate some comments to that effect.  If it isn't, then I would appreciate some comment explaning why the PyArg_ParseTuple(args, "O|O", &operation, &second_argument) isn't just PyArg_ParseTuple(args, "O", &operation).